docs/architecture/testing/driver-interface-test-strategy.md

```diff
@@ -4,9 +4,19 @@ This document describes the test strategy for the driver interfaces in Mbed Cryp
 
 The driver interfaces are standardized through PSA Cryptography functional specifications.
 
-## Secure element driver interface
+## Secure element driver interface testing
 
-The secure element driver interface (SE interface for short) is defined by [`psa/crypto_se_driver.h`](../../../include/psa/crypto_se_driver.h). This is an interface between Mbed Crypto and one or more third-party drivers.
+### Secure element driver interfaces
+
+#### Opaque driver interface
+
+The [unified driver interface](../../proposed/psa-driver-interface.md) supports both transparent drivers (for accelerators) and opaque drivers (for secure elements).
+
+Drivers exposing this interface needs to be registered at compile time by declaring their JSON description file.
```

Comment:
s/needs/need

Reply:
Fixed by amending the corresponding commit.

Reply:
Great.